### ISSUE TITLE
About temperature parameter

### ISSUE THREAD

**Issue by nonsensejoke** [bowen-gao/DrugCLIP#26]
Hi there, 

Thank you for developing this great model. I have a question related to the 'trainable temperature' parameter. 

In the following code, should we remove the `.detach()`? Or is it intentionally fixed? 

https://github.com/bowen-gao/DrugCLIP/blob/7a3a3fa33673f8668c811790f2e4681c98af44ef/unimol/models/drugclip.py#L197

Thank you very much for your time.
